packages/web-react/src/components/Navigation/useNavigationLinkProps.ts

```diff
@@ -0,0 +1,26 @@
+import { SpiritNavigationLinkProps } from '../../types';
+
+export type UseNavigationLinkProps = Partial<SpiritNavigationLinkProps> & {
+  'aria-current'?: string;
```

Comment:
I don't think we need it, this should be covered by some global props, shouldn't it? Because I can use aria-* or data-* everywhere.